src/main/java/org/time2java/tRussianBank/utils/ConvertTools.java

```diff
@@ -0,0 +1,15 @@
+package org.time2java.tRussianBank.utils;
+
+import org.time2java.tRussianBank.domain.User;
+import org.time2java.tRussianBank.domain.gaAnswer;
+
+import java.util.List;
+
+/**
+ * Created by time2die on 17.11.16.
+ */
+public class ConvertTools {
```

Comment:
\[Checkstyle\] ERROR: Utility classes should not have a public or default constructor\.